settings.gradle

```diff
@@ -0,0 +1 @@
+rootProject.name = 'screenshotTaker'
```

Comment:
User can clone repo ito folder with other name. [Gradle itself know path to project folder](https://stackoverflow.com/questions/41635131/gradle-getting-the-root-project-directory-path-when-starting-with-a-custom-buil/41635558).